app/controllers/api/v1/cfhs_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::V1::CfhsController < ApplicationController
+  def create
+    @cfh = Cfh.new cfh_params
+    if @cfh.save
+      render json: { message: "successful"}, status: 204
+    end
+  end
+
+private
+  def cfh_params
+    params.permit(:title, :message, :status)
+  end
+end
```

Comment:
You guys don't think you could call this something another programmer might possibly recognize?